content/altinn-studio/getting-started/app-dev-course/modul2/_index.nb.md

```diff
@@ -35,36 +35,34 @@ Datamodellen definerer hvilke data man forventer å samle inn, og hvilket format
 For et enkelt skjema er det ofte en 1-1 knytning fra felter i datamodellen til felter i skjemaet, mens for mer avanserte
 skjemaer kan datamodellen og skjemaet være ganske forskjellige.
 
-En datamodell med noen eksempel-felter følger med når man oppretter en tjeneste. Man kan redigere feltnavnene
+En datamodell med noen eksempelfelter følger med når man oppretter en tjeneste. Man kan redigere feltnavnene
 for å gjøre det tydelig hva de representerer, og legge til egne felter. 
 
 Datamodellen brukes til å bestemme _hva_ som skal hentes inn av data. Den brukes også til å bestemme _hvordan_ dataene
 skal se ut/være, ved at man setter på begrensninger som valideres.
 
 1. Naviger til "Datamodell"-siden ved å trykke på "Datamodell" i meny-linjen øverst på siden.
    - Du får da opp datamodellen, med de feltene som er lagt inn.
-2. Klikk på det første feltet i datamodellen, `property1`. Det dukker da opp en del redigeringsmuligheter i høyre-panelet.
+2. Klikk på det første feltet i datamodellen, `property1`. Det dukker da opp en del redigeringsmuligheter i panelet til høyre.
 3. Endre navnet på feltet `property1` til `fornavn`.
 4. Velg feltet `property2` ,og endre navnet på feltet til `etternavn`.
 5. Velg feltet `property3`, og endre navnet på feltet til `mellomnavn`.
 6. Merk at `mellomnavn`-feltet ikke er satt som påkrevd, ved at valget "Påkrevd" er skrudd av. Klikk på feltene 
     `fornavn` og `etternavn` og se at disse feltene er satt som påkrevd. Du kan beholde disse innstillingene som de er.
 7. Klikk på "Legg til"-knappen ved siden av `model` øverst i datamodellen og velg "Heltall".
-8. Se at det nå er lagt til et nytt felt med navn "name0" og type "Heltall" nederst i datamodellen. Endre navn på heltall-feltet til `alder`.
-9. Lenger nede i "Egenskaper"-panelet på høyre-siden er det et felt som heter "Minder eller lik". Skriv inn tallet `120` i dette feltet. 
+8. Se at det nå er lagt til et nytt felt med navn "name0" og type "Heltall" nederst i datamodellen. Endre navn på heltallfeltet til `alder`.
+9. Lenger nede i "Egenskaper"-panelet på høyresiden er det et felt som heter "Mindre enn eller lik". Skriv inn tallet `120` i dette feltet. 
 10. Klikk på "Legg til"-knappen ved siden av `model` øverst i datamodellen og velg "Objekt".
 11. Se at det nå er lagt til et nytt felt med navn "name0" og type "Objekt" nederst i datamodellen. Endre navn på objekt-feltet til `adresse`.
```

Comment:
Du ser at  det er lagt til et nytt felt med navnet "name0" og type "Objekt" nederst i datamodellen. Endre navnet på feltet til `Adresse`.

Reply:
Ville det ikke vært bedre å bruke futurum i tekster om hva leseren burde vente å se? Altså "du vil se at ..."? Det er i hvert fall beskrevet slik i artikkelen som det er lenket til fra saken; https://diataxis.fr/tutorials/#maintain-a-narrative-of-the-expected. Jeg tenker også at det gir et mer nøyaktig språk.

Reply:
Dette skal jeg ta opp i kurset vi skal ha - vi kan fint bruke presens til å uttrykke futurum, altså "blir" i stedet for "vil bli". Deter en enklere for for å uttrykke fremtid/hva du kommer til se/hva som kommer til å skje. Det gjør det lettere å skrive klart.